client/packages/system/src/Immunisation/ListView/ListView.tsx

```diff
@@ -26,26 +28,31 @@ const ImmunisationProgramListComponent: FC = () => {
   const {
     updateSortQuery,
     updatePaginationQuery,
-    queryParams: { sortBy, page, first, offset },
+    queryParams: { sortBy, page, first, offset, filterBy },
   } = useUrlQueryParams({ filters: [{ key: 'name' }] });
   const pagination = { page, first, offset };
   const navigate = useNavigate();
   const t = useTranslation('coldchain');
 
-  // later this will make api call
-  const draft: Program[] = [];
+  const queryParams = {
+    filterBy,
+    offset,
+    sortBy,
+    first,
+  };
+  const { data, isLoading, isError } = useImmunisationProgramList(queryParams);
 
   const columns = useColumns(
-    ['name', 'description'],
+    ['name'],
     {
       onChangeSortBy: updateSortQuery,
       sortBy,
     },
     [updateSortQuery, sortBy]
   );
 
-  // later create modal will use <InsertImmunisationProgram> type
-  const { isOpen, onClose, onOpen } = useEditModal<any>();
+  const { isOpen, onClose, onOpen } =
+    useEditModal<InsertImmunisationProgramInput>();
```

Comment:
As mentioned in chat, I get this the first time I click the new program button after refresh?

![Screenshot 2024-06-11 at 12 26 02 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/a7503108-c3cc-444a-a47d-88d7ea1fe9ac)

Removing the isOpen check below seems to fix... dunno why?